src/app/theme.ts

```diff
@@ -13,12 +13,13 @@ export const theme = {
     neutral: '#121521',
     neutral1: '#182027',
     neutral2: '#000e19',
+    callToAction: '#0e162b'
```

Comment:
callToAction nekoks pavadinims, is kart primet implementacija. Geriau gal tada primary tiesiog?